orcestra/flightplan.py

```diff
@@ -406,12 +406,116 @@ def expand_path(
 
     if performance := get_flight_performance(aircraft=aircraft):
         ds = ds.pipe(attach_flight_performance, performance)
-
     points_with_time = [(i, p) for i, p in enumerate(path) if p.time is not None]
+
     if len(points_with_time) > 1:
-        raise ValueError(
-            "Multiple waypoints have an associated time. Currently, only a single point with an associated time is implemented!"
+        i, point = points_with_time[0]
+        if point.time.tzinfo is None or point.time.tzinfo.utcoffset(point.time) is None:
+            warn(
+                f"Time {point.time} of {point} is naive (i.e. NOT timezone aware!). Assuming UTC."
+            )
+            reftime = np.datetime64(point.time)
+        else:
+            reftime = np.datetime64(
+                point.time.astimezone(datetime.timezone.utc).replace(tzinfo=None)
+            )
```

Comment:
This block seems to be repeated several times, so it could go into a function.

Reply:
Yes... I thought about that 😅 I'll change it